client/deployment/src/main/resources/templates/libraries/microprofile/api.qute

```diff
@@ -24,6 +24,12 @@ import io.quarkiverse.openapi.generator.annotations.GeneratedClass;
 import io.quarkiverse.openapi.generator.annotations.GeneratedMethod;
 import io.quarkiverse.openapi.generator.annotations.GeneratedParam;
 
+{#if use-bean-validation}
+
+import jakarta.validation.constraints.*;
+import jakarta.validation.Valid;
```

Comment:
To avoid [compilation errors due to ambiguity](https://github.com/quarkiverse/quarkus-openapi-generator/issues/103), can you please use FQCN rather than imports?